packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupDefault.stories.tsx

```diff
@@ -24,11 +24,13 @@ export const Default = (props: Partial<AvatarGroupProps>) => {
       {inlineItems.map(name => (
         <AvatarGroupItem name={name} key={name} />
       ))}
-      <AvatarGroupPopover>
-        {overflowItems.map(name => (
-          <AvatarGroupItem name={name} key={name} />
-        ))}
-      </AvatarGroupPopover>
+      {overflowItems && (
```

Comment:
why mess around with `undefined` values when you can just assert `overflowItems.length` ?

It saves you the extra worries of handling undefined assertions as well as accessing undefined. The validation on length is just as good

Reply:
It seems more straightforward and getting the warning `overflowItems may be undefined` will ensure the user takes that in mind. It also doesn't affect `AvatarGroup` directly, only when using `partitionAvatarGroupItems`.